src/overlay/OverlayManagerImpl.cpp

```diff
@@ -371,6 +371,16 @@ OverlayManagerImpl::connectToImpl(PeerBareAddress const& address,
                        "Peer rejected - all outbound pending connections "
                        "taken: {}",
                        address.toString());
+            CLOG_DEBUG(Overlay, "If you wish to allow for more pending "
```

Comment:
I think it would be nice to extract the logging here and in `addOutboundConnection` into a separate function to avoid duplication

Reply:
Good idea, I added a function `logPeerRejectionNoOutboundSlots` that deduplicates this.

Reply:
Ah, got it. I just made that change!